addons/web/static/src/legacy/scss/report.scss

```diff
@@ -13,6 +13,11 @@ body {
     font-family: $o-default-report-font;
 }
 
+hr {
+    border-top: 1px solid !important;
```

Comment:
this must be removed, it's already in bootstrap so it's not necessary, but also setting it as important you are possibly changing the value for people that have a different value (eg. custom theme).

Reply:
I've tested and it work for me (on the left without the fix, on the right with only the line `border-top-color: rgb(229, 229, 229);`)

![2023-03-23-144314_1920x1080_scrot](https://user-images.githubusercontent.com/9977887/227222589-632eb935-be67-4aa2-94ba-ca2311097792.png)